pkg/config/apiobjectstore.go

```diff
@@ -53,17 +53,45 @@ func init() {
 	scheme.AddKnownTypeWithName(secret.GroupVersionKind(), secret)
 }
 
+// ApiObjectStore is the database of the configured resources (Projects,
+// Registries and Scanners).
 type ApiObjectStore struct {
 	store      map[schema.GroupVersionKind][]runtime.Object
 	serializer *json.Serializer
+	path       string
+}
+
+// WriteManifest serializes the object specified by the obj parameter. The
+// filename parameter specifies the name of the file to be created. The path
+// where the file is created is determined the ApiObjectStore.
```

Comment:
I don't really understand this sentence. It means that the directory path is determined when we created the ApiObjectStore object, or we can determine the ApiObjectStore object with the path? (Or both of them, which means that it's OK :) )

Reply:
comment doc updated